src/components/user-steps-wrapper/UserStepsWrapper.jsx

```diff
@@ -56,7 +57,11 @@ const UserStepsWrapper = ({ userRole }) => {
           <StepProvider initialValues={initialValues} stepLabels={stepLabels}>
             <StepWrapper steps={stepLabels}>{childrenArr}</StepWrapper>
           </StepProvider>
-        )
+        ),
+        dialogConfig: {
+          title: titles.confirmTitle,
+          message: questions.unsavedChanges
```

Comment:
Add `const { t } = useTranslation()` in components and context.
```suggestion
          title: t(item.titles.confirmTitle),
          message: t(questions.unsavedChanges)
```